devices.js

```diff
@@ -8480,6 +8480,40 @@ const devices = [
             }
         },
     },
+    {
+        zigbeeModel: ['URC4450BC0-X-R'],
+        model: 'URC4450BC0-X-R',
+        vendor: 'Xfinity',
+        description: 'Alarm Security Keypad',
+        supports: 'action, arm',
+        meta: {configureKey: 1, battery: {voltageToPercentage: '3V_2100'}},
+        fromZigbee: [fz.command_arm, fz.temperature, fz.battery, fz.ias_occupancy_alarm_1, fz.identify, fz.ias_contact_alarm_1,
+            fz.ias_occupancy_alarm_2, fz.ias_occupancy_alarm_1_with_timeout],
+        exposes: [e.battery(), e.voltage(), e.occupancy(), e.battery_low(), e.tamper(), e.presence(),
+            exposes.numeric('action_code', 1), exposes.text('action_zone', 1), e.temperature(), e.action([
+                'disarm', 'arm_day_zones', 'identify', 'arm_night_zones', 'arm_all_zones', 'invalid_code', 'emergency',
+            ])],
+        toZigbee: [tz.arm_mode, tz.set_status],
```

Comment:
Can you control this device like documented here?  https://www.zigbee2mqtt.io/devices/3400-D.html#disarming

Reply:
Yes that works for disarming the keypad state internally, however this device requires the status (set_status) to be set in order for the indicator lights and beeping tones to be in sync with the system alarm states. I will create a page with the instructions once merged

Reply:
> Yes that works for disarming the keypad state internally, however this device requires the status (set_status) to be set in order for the indicator lights and beeping tones to be in sync with the system alarm states. I will create a page with the instructions once merged

@dettrick, have you done the status (set_status) documentation? Where can I find it? I am looking at a Linkind Keypad and has a similar behavior for a status update once it detects presence.